fcl/protodunevd/gen/gen_protodunevd_allradio.fcl

```diff
@@ -0,0 +1,98 @@
+#include "services_refactored_pdune.fcl"
+#include "singles_dune.fcl"
+#include "corsika_protodune.fcl"
+#include "CORSIKAGendp.fcl"
+#include "protodunevd_radiological_model_decay0.fcl"
+
+process_name: SinglesGen
+
+services:
+{
+  # Load the service that manages root files for histograms.
+  TFileService: { fileName: "gensingle_protoDUNE_hist.root" }
+  TimeTracker:       @local::dune_time_tracker
+  MemoryTracker:     @local::dune_memory_tracker
+  RandomNumberGenerator: {} #ART native random number generator
+  FileCatalogMetadata:  @local::art_file_catalog_mc
+  @table::protodunevd_refactored_simulation_services
+}
+
+#Start each new event with an empty event.
+source:
+{
+  module_type: EmptyEvent
+  timestampPlugin: { plugin_type: "GeneratedEventTimestamp" }
+  maxEvents:   100
+  firstRun:    1           # Run number to use for this file
+  firstEvent:  1           # number of first event in the file
+}
+
+# Define and configure some modules to do work on each event.
+# First modules are defined; they are scheduled later.
+# Modules are grouped by type.
+physics:
+{
+
+ producers:
+ {
+   ar39:             @local::protodunevd_39Ar_in_LAr
+   kr85:             @local::protodunevd_85Kr_in_LAr
+   ar42:             @local::protodunevd_42Ar_in_LAr
+   k42fromar42:      @local::protodunevd_42Kfrom42Ar_in_LAr
+   k40cathode1:      @local::protodunevd_40K_in_Cathode_nonTCOside
```

Comment:
Generators may be simplified? See comment on https://github.com/DUNE/dunesim/pull/80